src/h2olog/h2olog.cc

```diff
@@ -230,6 +265,13 @@ int main(int argc, char **argv)
         exit(EXIT_FAILURE);
     }
 
+    if (list_and_exit) {
+        for (const auto &usdt : tracer->usdt_probes()) {
+            fprintf(stderr, "%s:%s\n", usdt.provider.c_str(), usdt.name.c_str());
```

Comment:
I think the result should go into stdout, as this is the normal behavior (that ends up in `EXIT_SUCCESS`)?